Automattic/woocommerce-payments
# Move the Payments → Overview task list above account details card to improve visibility

Fixes #4399

#### Changes proposed in this Pull Request





This PR moves the **Things to do** task list above the **Account Details** card on the **Payments → Overview** screen to improve the visibility of disputes needing a response.





#### Testing instructions



1. Using [WCPay Dev Tools](https://github.com/Automattic/woocommerce-payments-dev-tools), ensure the `Enable account overview task list` checkbox is checked.
2. Purchase a product with a disputed payment method (`4000000000000259`).
3. Go to the **Payments → Overview** screen.
4. The **Things to **do task list should be located above the **Account Details**card.



**Before**
<img width="500" alt="image" src="https://user-images.githubusercontent.com/3147296/177257459-c78276f8-9f6a-43ba-8f3b-97e3b0557591.png">


**After**
<img width="500" alt="image" src="https://user-images.githubusercontent.com/3147296/177257538-a9cf8f3b-c2c3-40a9-90e0-6f37a1bb06cd.png">


-------------------


**Post merge**
